content/en/docs/Configuration/p8s-jaeger-grafana/prometheus.md

```diff
@@ -0,0 +1,183 @@
+would be very rare to see data delays, but should you notice any delays you may
+tune caching parameters to values that work better for your environment. These
+are the default values:
+
+```yaml
+spec:
+  external_services:
+    prometheus:
+      cache_enabled: true
+      # Per-query expiration in seconds
+      cache_duration: 10
+      # Global cache expiration in seconds. Think of it as
+      # the "reset" or "garbage collection" interval.
+      cache_expiration: 300
+```
+
+### Compatibility with Prometheus-like servers
+
+Although Kiali assumes a Prometheus server and is tested against it, there are
+<abbr title="Time series databases">TSDBs</abbr> that can be used as Prometheus
+replacement despite not implementing the full Prometheus API. 
+
+Community users have faced two issues when using Prometheus-like TSDBs:
+* Kiali may report that the TSDB is unreachable, and/or
+* Kiali may show empty metrics if the TSBD does not implement the `/api/v1/status/config`.
+
+To fix these issues, you may need to provide a custom health check endpoint for
+the TSDB and/or manually provide the configurations that Kiali reads from the
+`/api/v1/status/config` API endpoint:
+
+```yaml
+spec:
+  external_services:
+    prometheus:
+      # Fix the "Unreachable" metrics server warning.
+      health_check_url: "http://custom-tsdb-health-check-url"
+      # Fix for the empty metrics dashboards
+      thanos_proxy:
+        enabled: true
+        retention_period: "7d"
+        scrape_interval: "30s"
+```
+
+## Prometheus Tuning
+
+Production environments should not be using the Istio Prometheus add-on, or carrying over its configuration settings.  That is useful only for small, or demo installations.  Instead, Prometheus should have been installed in a production-oriented way, according to [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/installation).
+
+This section is primarily for users where Prometheus is being used specifically for Kiali, and possible optimizations that can be made knowing that Kiali does not utilize all of the default Istio and Envoy telemetry.
+
+
+### Metric Thinning
+
+Istio and Envoy generate a large amount of telemetry for analysis and troubleshooting.  This can result in significant resources being required to ingest and store the telemetry, and to support queries into the data.  If you use the telemetry specifically to support Kiali, it is possible to drop unnecessary metrics and unnecessary labels on required metrics.  This [FAQ Entry]({{< ref "/docs/faq/general#requiredmetrics" >}}) displays the metrics and attributes required for Kiali to operate.
+
+To reduce the default telemetry to only what is needed by Kiali[^1] users can add the following snippet to their Prometheus configuration. Because things can change with different versions, it is recommended to ensure you use the correct version of this documentation, based on your Kiali/Istio version.
```

Comment:
```suggestion
To reduce the default telemetry to only what is needed by Kiali[^1] users can add the following snippet to their Prometheus configuration. Because things can change with different versions, it is recommended to ensure you use the correct version of this documentation based on your Kiali/Istio version.
```